lib/cabal.project

```diff
@@ -0,0 +1,15 @@
+packages:
+    ./asset/manifest
+    ./asset/serve-snap
+    ./backend
+    ./cliapp
+    ./command
+    ./executable-config/inject
+    ./executable-config/lookup
+    ./frontend
+    ./route
+    ./run
+    ./selftest
+    ./snap-extras
+    ./tabulation
+
```

Comment:
Adding something like:
```
package *
  ghc-options: -Wwarn
```
is necessary for newer GHC versions, since we have `-Werror` for unneeded imports, and cabal fails for that reason if you try to open, let's say 'selftest'.

Reply:
Added it to the cabal.project for good, since otherwise, it is not useful.